modules/repository-datastax/src/main/resources/com/intuit/wasabi/repository/impl/cassandra/migration/V035__Create_ExperimentTag_table.cql

```diff
@@ -0,0 +1,5 @@
+create table experiment_tag (
+  app_name text,
+  tags set<text>,
```

Comment:
This table stores the experiment tags per application, that's why I called it experiment_tag :)